bpo-28356: Document os.rename() behavior on Windows for differing filesystems (#27376)
python/cpython: Misc/NEWS.d/next/Documentation/2021-07-26-19-57-21.bpo-28356._F5a04.rst

> The documentation about os.rename() used on Windows systems updated to reflect behavior about moving across filesystems. Patch by Ryan Ozawa

Reviewer: ```suggestion
Document the behavior of :func:`os.rename` when moving across filesystems on Windows. Patch by Ryan Ozawa.
```
Shorter, and NEWS entries should be in the imperative
